codalab/lib/codalab_manager.py

```diff
@@ -403,11 +403,9 @@ def auth_handler(self, mock=False):
 
         if mock or handler_class == 'MockAuthHandler':
```

Comment:
Can we get rid of this too, since now you have to connect to the rest server to do anything, so MockAuthentation doesn't seem like it's useful to have.